src/index.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { render } from 'react-dom'
+
+class App extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      givenTemp: 82,
+      convertToCelsius: true
+    }
+
+    this.toggleConversion = this.toggleConversion.bind(this)
+    this.updateGivenTemp = this.updateGivenTemp.bind(this)
+    this.convert = this.convert.bind(this)
```

Comment:
My editor kept complaining that it would only allow that in TypeScript. It was annoying so I just switched the implementation to shut it up.